src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,60 +1,52 @@
 import React from 'react';
-import { Movie } from '../../types/Movie';
 
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
 type Props = {
-  movie: Movie;
+  movie: Movie
 };
 
-export const MovieCard: React.FC<Props> = ({ movie }) => {
-  const {
-    title,
-    description,
-    imgUrl,
-    imdbUrl,
-  } = movie;
-
-  return (
-    <div className="card" data-cy="Movie">
-      <div className="card-image">
-        <figure className="image is-4by3">
-          <img
-            data-cy="MovieImage"
-            src={imgUrl}
-            alt="Film logo"
-          />
-        </figure>
-      </div>
-
-      <div className="card-content">
-        <div className="media">
-          <div className="media-left">
-            <figure className="image is-48x48">
-              <img src="images/imdb-logo.jpeg" alt="imdb" />
-            </figure>
-          </div>
+export const MovieCard: React.FC <Props> = ({ movie }) => (
+  <div key={movie.imdbId} className="card" data-cy="Movie">
```

Comment:
Remember to set the key prop where you add the component, rather than inside it.